### DISCUSSION TITLE
Am I missing the big picture possibly?

### DISCUSSION THREAD

**Discussion by Buddyboy-git** [saleor/saleor#10720] [General]
Im trying to get back into web programming and starting with an ecommerce site. Its been a while since i have coded anything. Before I pick a final shopping cart solution for me to modify, I am looking at Saleor and python as an option.  

I currently have opencart installed on my web host. No matter what ecommerce package I choose it will need a checkout module overhaul because all shopping carts are designed for shipping/dropshipping. I will be running a brick and mortar, pickup/delivery market place. 

My Experience so far...
1. I failed miserably to install saleor on my windows laptop. Using Docker, and manually.
2. I tried to install on my host. but It seems you need a dedicated server for hosting all these packages. And that immediately drives up the hosting costs.
3. I created a sandbox developer account on saleor.io and so far I'm not impressed with the dashboard.   

My questions...
1. Where are the instructions to access my saleor cloud storefront as an enduser? what is the url?
2. Are there any existing websites that are actually using saleor?  I would like to see if it is possible to put lipstick on this thing.  It is definitely raw right out of the gate.

Thanks,

**Comment by peelar** [comment]
Hello @Buddyboy-git 👋 

> My Experience so far...
I am sorry to hear that the experience you've had was not what you expected. For local development, did you use [saleor-platform](https://github.com/saleor/saleor-platform)? If so, can you file an issue for what was not working?

> My questions...
1. Storefront is not an application we set up for you by default. If you want to generate one for your Saleor Cloud API, the easiest way to do it would be [using Saleor CLI](https://docs.saleor.io/docs/3.x/cli#storefront).
2. Yes, many production-grade applications of various scales run on Saleor. Unfortunately, we can't share the details because of legal obligations. Please head over to [demo.saleor.io](https://demo.saleor.io/default-channel/en-US/) if you want to see Saleor in action (with Storefront + Checkout).

Please let me know if you have any other questions. Have a nice day.
